RadixWallet/Features/InteractionReview/Sections/Sections+ExecutionSummary.swift

```diff
@@ -41,7 +45,27 @@ extension InteractionReview.Sections {
 
 		switch summary.detailedManifestClass {
 		case nil:
-			return nil
+			guard interactionId?.isWalletAccountDeleteInteraction == true else { return nil }
```

Comment:
this check can go away, and just use the manifest classification that will be merged in Sargon.